website/content/docs/secrets/databases/snowflake.mdx

```diff
@@ -70,38 +48,174 @@ the root credentials and still be able to access your account should something g
         password="vaultpass"
     ```
 
-1.  Configure a role that maps a name in Vault to an SQL statement to execute to
-    create the database credential:
+    A properly formatted data source name (DSN) needs to be provided during configuration of the
+    database. This DSN is typically formatted with the following options:
+
+    ```shell-session
+    {{username}}:{{password}}@account/db_name
```

Comment:
Does the DSN need to be changed if using RSA credentials? Or is this still correct?

Reply:
Nope, no change needed to the DSN for this (until there is demand to use key pair authn for the root credential given to Vault :)).